proto/c4echain/cfeminter/tx.proto

```diff
@@ -1,13 +1,35 @@
 syntax = "proto3";
 package chain4energy.c4echain.cfeminter;
 
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "c4echain/cfeminter/params.proto";
+import "c4echain/cfeminter/minter.proto";
+
 // this line is used by starport scaffolding # proto/tx/import
 
 option go_package = "github.com/chain4energy/c4e-chain/x/cfeminter/types";
 
 // Msg defines the Msg service.
 service Msg {
```

Comment:
// Don't see the point of separated request for changing denom without changing minter
// rather one message for changing all params
// and one message to change minters (with start time) as changing denom on working blockchain is not a common use case
// and also going with naming rules:
// MsgUpdateMinters should be changed to MsgUpdateMintersParams, MsgUpdateParams should be added, MsgUpdateMintDenom should be removed